src/utils/Interfaces.ts

```diff
@@ -408,4 +408,32 @@ export interface UYAPlayerDetails {
     deathmatch_stats: UyaDeathmatchStatsSchema;
     siege_stats: UyaSiegeStatsSchema;
     ctf_stats: UyaCTFStatsSchema;
+}
+
+
+// Type definitions for players and games data
+export interface UYAOnlinePlayer {
+    username: string;
+};
+
+export interface UYAOnlinePlayerResponse {
+    count: number;
+    results: UYAOnlinePlayer[];
+}
+
+export interface UYAOnlineGame {
+    name: string;
+    game_status: string;
```

Comment:
What are the string variants for this? I saw `WorldStaging` in the screenshots. In TS, you can specify a set of strings instead of `string`. That might be better for documentation purposes.
    ... 
   game_status = "WorldStatus" | "InGame" | "PostGame";
    ...